### PR TITLE
Add BART finetuned on CNN+DM for summarisation

### PR DESCRIPTION
**Pull request by abheesht17**

### CODE REVIEW

File: keras_nlp/models/bart/bart_presets.py [hunk 1]
```diff
@@ -71,3 +71,40 @@
         "merges_hash": "75a37753dd7a28a2c5df80c28bf06e4e",
     },
 }
+
+
+seq_2_seq_lm_presets = {
+    "bart_large_en_cnn": {
```
Comment (mattdangerw) [inline keras_nlp/models/bart/bart_presets.py:77]: Hmm, this is our first time facing this question, but I actually think we can just ship this as a backbone right? Because there are no new weights?

I don't see any disadvantage, and it would have the advantage of being able to load this on a backbone and run absolutely any workflow on top (e.g. your own "task flow" with custom code if you so chose).

**Review by mattdangerw — APPROVED** [review]
Code looks good to me. Does it need updates (e.g. hash) for the new weights I just uploaded to the bucket?

**Comment by abheesht17** [comment]
> Code looks good to me. Does it need updates (e.g. hash) for the new weights I just uploaded to the bucket?

Already done!

**Comment by mattdangerw** [comment]
> The failed accelerator test is because of DistilBERT, apparently.

Ah probably that's -> https://github.com/keras-team/keras-nlp/issues/960

Still need to look into that.

Thanks!
